qiita_pet/support_files/doc/source/tutorials/prepare-templates.rst

```diff
@@ -37,6 +37,8 @@ These are the columns required for successfully submit your data to EBI:
 | ``physical_specimen_location``   | free text            | Where you would go to find physical sample or DNA, regardless of whether it is still available or not.   |
 +----------------------------------+----------------------+----------------------------------------------------------------------------------------------------------+
 | ``taxon_id``                     | integer              | NCBI's taxon id for the sample                                                                           |
++-------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------+
+| ``description``                  | free text            | Description of the sample.                                                                               |
```

Comment:
The rendering is a little bit off, would you mind having a look at the formatting?

![screen shot 2015-10-09 at 10 09 59 am](https://cloud.githubusercontent.com/assets/375307/10400509/01e1183c-6e6e-11e5-977e-9ffe1ca69770.png)